### PR TITLE
add blog

### PR DESCRIPTION
**Pull request by H3lloTom**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.
e.g.

``` md
---
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [x] Add `<!--more-->` section to just show the abstract in blog homepage.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.github.io#contribute-guideline)

### CODE REVIEW

File: _posts/2020-06-02-wechat-bot.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+title: "wechat-bot"
```
Comment (lijiarui) [inline _posts/2020-06-02-wechat-bot.md]: Please add a more clear title for this blog, then others would like to click and read your blog.

**Review by lijiarui — CHANGES_REQUESTED** [review]
Good job! Please change as I request and I would merge this PR.

File: _posts/2020-06-02-wechat-bot.md (hunk 1 above)
Comment (H3lloTom) [reply _posts/2020-06-02-wechat-bot.md]: ok，glad to get your replay  (●'◡'●)

File: _posts/2020-06-02-wechat-bot.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+---
+title: "优惠券搜索机器人（Coupon wechat bot）"
+author: h3llotom
+date: "2020-06-02 12:00:00 +0800"
+categories: project
+tags:
+  - wechaty
+  - wechaty-puppet-padplus
+header:
+  teaser: /assets/2020/2020-06-02-wechat-bot.jpg
```
Comment (lijiarui) [inline _posts/2020-06-02-wechat-bot.md]: Please create a new folder under `/assets/2020` and put the image in the new folder. Thanks!

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: _posts/2020-06-02-wechat-bot.md (hunk 2 above)
Comment (H3lloTom) [reply _posts/2020-06-02-wechat-bot.md]: done~

**Comment by lijiarui** [comment]
Good job！

**Review by lijiarui — APPROVED** [review]

**Pull request merged**